ReadMe.md

```diff
@@ -79,3 +79,6 @@ Follow these steps for completing your project.
 - [ ] Submit a pull request to merge <firstName-lastName> Branch into master (student's  Repo). **Please don't merge your own pull request**
 - [ ] Add your Team Lead as a reviewer on the pull request
 - [ ] Your Team Lead will count the project as complete by merging the branch back into master
+
+## Future Todo
+- [ ] Smaller image sizes for faster loading.
```

Comment:
There are lots of ways to optimize images:
* Reduce the image dimensions
* [Responsive Images](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images) - Create the image in several different sizes and let the browser decide which to send. Phones get smaller one, 4k desktop monitors get bigger ones.
* [Reduce the data itself (PNG only)](https://tinypng.com/) - PNG has some optimization capabilities that can reduce a file up to like 66% without noticing a quality loss.
* [Use progressive loading (JPG only)](https://www.jpeg.io/) - The user will see a low-quality image that slowly gets better, instead of scanning top to bottom, which means they can start using the image long before the full download. __Note: This may slightly increase file size, but it significantly improves the perception of performance.__

Reply:
I did reduce all the image sizes since then, and we'll be learning about responsive design starting next week. Resizing the images to the same dimensions also helped with much of the messy code I was struggling with. Looking forward to responsive design. Good to know about the PNG & I'll have to remember progressive loading.